src/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -463,6 +498,9 @@ else if (hasDocument(messageRecord)) {
       documentViewStub.get().setDocument(new DocumentSlide(context, messageRecord));
       documentViewStub.get().setDocumentClickListener(new ThumbnailClickListener());
       documentViewStub.get().setOnLongClickListener(passthroughClickListener);
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+        documentViewStub.get().setImportantForAccessibility(View.IMPORTANT_FOR_ACCESSIBILITY_NO_HIDE_DESCENDANTS);
```

Comment:
why can't that be set in the corresponding `.xml`? (`document_view.xml` or `conversation_item_*.xml`)

same for the other types

Reply:
because it is only supported for >=kitkat and I am not sure what will happen if I set an "invalid" value there, and duplicating layout files in a layout-vXX folder would be more inconvenient